src/main/java/frc/robot/commands/FrictionBrakeRelease.java

```diff
@@ -0,0 +1,32 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.Climbers;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class FrictionBrakeRelease extends InstantCommand {
+  Climbers m_climbers;   
+
+  public FrictionBrakeRelease(Climbers subsystem) {
```

Comment:
You shouldn't need to add this method. The code that runs an InstantCommand would be in the initialize method.  In regular Commands, it would be in the execute.